src/apps/schema/src/appRevamp/components/AddFieldModal/FieldFormInput.tsx

```diff
@@ -251,6 +327,13 @@ export const FieldFormInput = ({
               null
             }
             options={dropdownOptions}
+            renderOption={(props: any, option: DropdownOptions) => {
+              return (
+                <Box {...props} key={option.value}>
```

Comment:
The error that I'm getting is because we have duplicate names on some of the folder labels. 
And upon checking, the other autocomplete fields still work properly.